src/lib/dma.c

```diff
@@ -116,9 +116,9 @@ struct dma *dma_get(uint32_t dir, uint32_t cap, uint32_t dev, uint32_t flags)
 
 	dmin->sref++;
 
-	tr_info(&dma_tr, "dma_get() ID %d sref = %d busy channels %ld",
-		dmin->plat_data.id, dmin->sref,
-		atomic_read(&dmin->num_channels_busy));
+	tr_dbg(&dma_tr, "dma_get() ID %d sref = %d busy channels %ld",
```

Comment:
and this

Reply:
@btian1  Please don't mark as resolved without addressing the comment.

Reply:
this should be missed during push, let me change it back.

Reply:
> ed to map the Zephyr log levels to legacy at some point.
> @btian1 I would update this PR to o

Thanks, latest version already include agreed log, and for logs under discussion, I already reverted back to origin.
Hopefully, this PR can be merged before your holiday, @kv2019i @lgirdwood